app/uk/gov/hmrc/lisaapi/connectors/DesConnector.scala

```diff
@@ -41,28 +41,28 @@ class DesConnector @Inject()(
                               runMode: RunMode
                             )(implicit ec: ExecutionContext) extends ServicesConfig(runModeConfiguration: Configuration, runMode: RunMode) {
```

Comment:
Out of Scope - I don't think ServicesConfig should be extended here, if they need to pull something from config, it should be done in AppContext which already gets injected here.

Reply:
refactered this, saves it being missed in the future